scss/_patterns_navigation.scss

```diff
@@ -3,6 +3,16 @@
 $navigation-hover-opacity: 0.58;
 
 @mixin vf-p-navigation {
+  %navigation-row {
+    @extend %fixed-width-container;
```

Comment:
do we need to override the max width on line 141? Why not take `@extend %fixed-width-container;` outside of the %navigaiton-row placeholder, and add it on line 138? this way there will be no redefinition of max-width

Reply:
@lyubomir-popov Good point. Will try that.

Reply:
@lyubomir-popov Unfortunately, wont work. %fixed-width-container defines more than just max-width. It comes with margins, etc.
So if we don't extent it in `--full-width` variant, we would need to redefine width: 100%, margin: auto, etc that it provides.